src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderEmploymentRepositoryTest.java

```diff
@@ -58,7 +60,9 @@
         .postType(new EmploymentStatus("FT", "Full Time"))
         .employerName("HGuvZZoLHGuvZZo")
         .supervisorName("MarMar")
-        .occupation(new Occupation("1204", "Builder"))
+        .occupation(new Occupation("5807", "Driver"))
+        .scheduleType(new EmploymentSchedule("INHAND", "In Hand"))
```

Comment:
The updated mock data in src/main/resources/db/migration/data/R__4_35__OFFENDER_EMPLOYMENTS.sql  covers the extra deserialisation cases.